frontend/www/js/omegaup/components/identity/Editv2.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div class="omegaup-edit-identity card">
```

Comment:
```suggestion
  <div class="card" data-edit-identity>
```

si se necesita para las preubas, si no, es mejor eliminarlo.

Reply:
Parece que no se utiliza para nada, así que se va